frontend/src/components/ResultsList.js

```diff
@@ -3,7 +3,7 @@ import '../sass/components/resultslist.scss';
 import '../sass/base/typography.scss';
 import Result from './Result';
 
-const sampleJSON = {
+const appIdeas= {
```

Comment:
It will be better if `appIdeas` is an array of objects rather than an object of objects. This will lead to better performance, see explanation below.